custom_components/tuya_local/devices/breville_smartairconnect_purifier.yaml

```diff
@@ -105,7 +92,7 @@ secondary_entities:
       - id: 20
         name: sensor
         type: integer
-        unit: min
+        unit: d
```

Comment:
It looks like you changed the wrong one here.  This was correct at "min", I think.  The one to change to "d" should be the "days" one.  Also, I suggested adding class: duration there as well (this one already has it).

Reply:
Whoops, you're right, days make no sense there. I tried setting the Filter days left with `class: duration` and the `d` unit. The result is technically true (4128:00:00 is 172 days), but may not be immediately interpretable to the user (see screenshot below).
![smart_air_connect_filter_days_w_duration](https://user-images.githubusercontent.com/6414258/221353520-3cda249c-85a4-4611-99c1-3733d3d671dc.jpg)

The config for this was:
```
  - entity: sensor
    name: Filter days left
    category: diagnostic
    class: duration
    icon: "mdi:air-filter"
    dps:
      - id: 16
        name: sensor
        type: integer
        unit: d
```

Are there other advantages to specifying the class as duration (e.g. in different themes)? Or is it better to drop `class: duration` and specify the unit as `days` for readability?

Reply:
I wasn't aware that duration only displayed up to hours. I thought it would use a sensible unit for the scale of the current value, which in this case might be months or weeks. If it only displays h:m:s, then it would be better without the class.

Reply:
No worries, I've reverted the `unit: d` change on the Time left sensor.